cmd/du.go

```diff
@@ -0,0 +1,130 @@
+/*
+This file is part of REANA.
+Copyright (C) 2022 CERN.
+
+REANA is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"os"
+	"reanahub/reana-client-go/client"
+	"reanahub/reana-client-go/client/operations"
+	"reanahub/reana-client-go/utils"
+	"reanahub/reana-client-go/validation"
```

Comment:
When I saved the file for me the imports where automatically sorted like this:
```
	"fmt"
	"os"
	"reanahub/reana-client-go/client"
	"reanahub/reana-client-go/client/operations"
	"reanahub/reana-client-go/utils"
	"reanahub/reana-client-go/validation"

	"github.com/spf13/cobra"
```
Not sure if it's some kind of external plugin which comes with go extension in VS Code, or what. We might need to figure this out in the future and align these kind of things to make it consistent in all the files

Reply:
I think I found the problem, it's `goimports` that was probably activated for you and not for me. We can include it in the `golangci-lint` to make it consistent for everyone

Reply:
I added it to the lint checks. See if you like it

Reply:
Yeah, looks nice, thanks for a quick action!